src/authorization/dynamic-data-source/AssignedSiteAreasDynamicAuthorizationDataSource.ts

```diff
@@ -32,13 +32,16 @@ export default class AssignedSiteAreasDynamicAuthorizationDataSource
       ['id']
     );
     const siteIDs = _.uniq(_.map(sites.result, 'id'));
-    const siteAreas = await SiteAreaStorage.getSiteAreas(this.tenant.id,
-      {
-        siteIDs: siteIDs,
-        issuer: true,
-      }, Constants.DB_PARAMS_MAX_LIMIT,
-      ['id']
-    );
-    return _.uniq(_.map(siteAreas.result, 'id'));
+    if (siteIDs.length !== 0) {
```

Comment:
I think the whole AssignedSiteAreas dynamic filter should be removed... seems like everything works just fine with AssignedSites filter. What do you think? Should I completely remove it? Or do you think it might be needed in the future? (It is currently not used anywhere anymore.)

Reply:
> I think the whole AssignedSiteAreas dynamic filter should be removed... seems like everything works just fine with AssignedSites filter. What do you think? Should I completely remove it? Or do you think it might be needed in the future? (It is currently not used anywhere anymore.)

Not used -> Removed :-)
Indeed, you can use the AssignedSites instead.

Reply:
Great, I removed everything related to that. :)